packages/runtime/test/integration/group-inner-transaction.ts

```diff
@@ -0,0 +1,75 @@
+import { types } from "@algo-builder/web";
+import { assert } from "chai";
+
+import { Runtime } from "../../src";
+import { AccountStoreI, AppInfo } from "../../src/types";
+import { useFixture } from "../helpers/integration";
+
+describe("Group inner transaction", function () {
+	useFixture("group-inner-transaction");
+
+	let runtime: Runtime;
+	let appInfo: AppInfo;
+	let alice: AccountStoreI;
+	let contractAcc: AccountStoreI;
+
+	function syncAccounts(): void {
+		contractAcc = runtime.getAccount(appInfo.applicationAccount);
+		alice = runtime.getAccount(alice.address);
```

Comment:
```suggestion
		alice = runtime.defaultAccounts();
```
Im pretty sure this should work exactly the same